packages/api/index.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 exports.token = require('./endpoints/token');
+exports.googleToken = require('./endpoints/googleToken').handler;
```

Comment:
If we move the auth logic to the lib folder, we can keep using the same token endpoint for both google and earthlogin. This is also super useful when we have to implement the Launchpad auth in the next PI